src/test/unit/pid_unittest.cc

```diff
@@ -410,8 +417,7 @@ TEST(pidControllerTest, testPidLevel)
 
     // Disable ANGLE_MODE
     disableFlightMode(ANGLE_MODE);
-    currentPidSetpoint = pidLevel(FD_ROLL, pidProfile, &angleTrim, currentPidSetpoint, calcHorizonLevelStrength());
-    EXPECT_FLOAT_EQ(11.07958, currentPidSetpoint);
+    currentPidSetpoint = pidLevel(FD_ROLL, pidProfile, &angleTrim, currentPidSetpoint, calcHorizonLevelStrength()
```

Comment:
```suggestion
    currentPidSetpoint = pidLevel(FD_ROLL, pidProfile, &angleTrim, currentPidSetpoint, calcHorizonLevelStrength());
    EXPECT_FLOAT_EQ(???, currentPidSetpoint);
```

It might be possible that the next failing test will tell you what the new expected value needs to be.